kivy/tests/test_window_base.py

```diff
@@ -126,3 +127,19 @@ def test_window_opacity_clamping_negative(self):
         if self.check_opacity_support():
             self.Window.opacity = -1.5
             self.assertEqual(self.Window.opacity, 0.0)
+
+
+class WindowNativeHandleTest(GraphicUnitTest):
```

Comment:
I guess another property of native handles is that they should not change over time, but I really don't know how to exercise that in a test.

Reply:
Checking if we're getting a value when we expect one is fine for me ATM.